LrcParser.Tests/Parser/Lrc/Utils/LrcTimedTextUtilsTest.cs

```diff
@@ -12,26 +12,61 @@ public class LrcTimedTextUtilsTest
+    [TestCase("帰り道は", 0, "帰り道は", new string[] { })]
+    [TestCase("", 0, "", new string[] { })]
+    [TestCase("   ", 0, "", new string[] { })]
+    [TestCase(null, 0, "", new string[] { })]
+    // Surrounding time tags
+    [TestCase(
+        "<00:06.84> Every <00:07.20>   <00:07.56> night <00:07.87>   <00:08.19> that <00:08.46>   <00:08.79> goes <00:09.19>   <00:09.59> between", 6840,
+        "Every night that goes between",
+        new[] { "[0,start]:6840", "[4,end]:7200", "[6,start]:7560", "[10,end]:7870", "[12,start]:8190", "[15,end]:8460", "[17,start]:8790", "[20,end]:9190", "[22,start]:9590" }
+    )]
+    // Alternating time tags, spaced on both sides
+    [TestCase(
+        "<00:06.84> Every <00:07.56> night <00:08.19> that <00:08.79> goes <00:09.59> between", 6840, "Every night that goes between",
+        new[] { "[0,start]:6840", "[6,start]:7560", "[12,start]:8190", "[17,start]:8790", "[22,start]:9590" }
+    )]
+    // Alternating time tags, unspaced
+    [TestCase(
+        "<00:06.84>Every<00:07.56>night<00:08.19>that<00:08.79>goes<00:09.59>between", 6840, "Everynightthatgoesbetween",
+        new[] { "[0,start]:6840", "[5,start]:7560", "[10,start]:8190", "[14,start]:8790", "[18,start]:9590" }
+    )]
+    [TestCase(
+        "Every<00:07.56>night<00:08.19>that<00:08.79>goes<00:09.59>between", 6840, "Everynightthatgoesbetween",
+        new[] { "[0,start]:6840", "[5,start]:7560", "[10,start]:8190", "[14,start]:8790", "[18,start]:9590" }
+    )]
+    // Alternating time tags, prefix spaced
+    [TestCase(
+        "<00:06.84> Every<00:07.56> night<00:08.19> that<00:08.79> goes<00:09.59> between", 6840, "Every night that goes between",
+        new[] { "[0,start]:6840", "[6,start]:7560", "[12,start]:8190", "[17,start]:8790", "[22,start]:9590" }
+    )]
+    [TestCase(
+        "Every<00:07.56> night<00:08.19> that<00:08.79> goes<00:09.59> between", 6840, "Every night that goes between",
+        new[] { "[0,start]:6840", "[6,start]:7560", "[12,start]:8190", "[17,start]:8790", "[22,start]:9590" }
+    )]
+    // Alternating time tags, postfix spaced
+    [TestCase(
+        "<00:06.84>Every <00:07.56>night <00:08.19>that <00:08.79>goes <00:09.59>between", 6840, "Every night that goes between",
+        new[] { "[0,start]:6840", "[6,start]:7560", "[12,start]:8190", "[17,start]:8790", "[22,start]:9590" }
+    )]
+    [TestCase(
+        "Every <00:07.56>night <00:08.19>that <00:08.79>goes <00:09.59>between", 6840, "Every night that goes between",
+        new[] { "[0,start]:6840", "[6,start]:7560", "[12,start]:8190", "[17,start]:8790", "[22,start]:9590" }
+    )]
+    // Multiple empty tags
+    [TestCase(
+        "<00:51.00> <01:29.99><01:48.29>  <02:31.00> <02:41.99>You gotta fight !", 0,
+        "You gotta fight !", new[] { "[0,start]:161990" }
+    )]
+    public void TestDecode(string text, int lineStartTime, string expectedText, string[] expectedTimeTags)

```

Comment:
Remove the lineStartTime because all the actual value should(and will) be zero.

Reply:
You mean just for this test? Should I split the tests where it's zero, maybe?

Reply:
All test cases within this test lineStartTime should be 0.
You can make another test with few test case to test the result if lineStartTime is not zero if needed.

Reply:
Ok, then I'll split it :+1:
Should also decrease the number of changed lines on the existing test.

Reply:
[Fixed](https://github.com/karaoke-dev/LrcParser/compare/c57f5e128cb7287b42443a7b0e611e0597dff2f8..81381f75679d8326ea6f7a88aba684c8cce8addb).